### PR TITLE
Add admin-guide for hubble

### PR DESCRIPTION
**Pull request by chowbao**
First pass at adding an `admin-guide` section for hubble. Includes general information about:

* prerequisites
* installing
* configuring
* running

Where it mostly links to other existing documentation within the `stellar-etl-airflow` README docs and will leverage the Ledger Exporter docs whenever they are written instead of duplicating Ledger Exporter setup docs.

### CODE REVIEW

File: network/hubble/admin-guide/visualization/getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Getting Started 
+sidebar_position: 20
+---
+
+This sections goes through using [Google's Looker Studio](https://lookerstudio.google.com/u/0/navigation/reporting) as a free and easy to use visualization tool that you can hook up your BigQuery Stellar network data to. 
+
+There are many other free/paid visualization tools out there. Hubble data should be compatible with most if not all tools.
```
[the remark this answers is not in the document]
Comment (sydneynotthecity) [reply network/hubble/admin-guide/visualization/getting-started.mdx:8]: would also reword that any visualization tool with a BigQuery connector is compatible with Hubble.

File: network/hubble/admin-guide/visualization/getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+* Click `Data Source`
+* Find the `BigQuery` connector
+* Use the project `crypto-stellar`
+* Use the dataset `crypto_stellar`
+* Select the table of interest
+* Click `CONNECT`
+
+When you create a new report you should be able to now access data from `crypto-stellar.crypto_stellar.<table name>`
+
+## Making your first pie chart
```
Comment (sydneynotthecity) [inline network/hubble/admin-guide/visualization/getting-started.mdx:28]: YOU DIDN'T

File: network/hubble/admin-guide/scheduling-and-orchestration/getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+:::
+
+## Configuring GCP Composer Airflow
+
+There are two things required for the configuration and setup of GCP Composer Airflow:
+
+* Upload DAGs to the Composer Airflow Bucket
+* Configure the Airflow variables for your GCP setup
+
+For more detailed instructions please see the [stellar-etl-airflow Installation and Setup](https://github.com/stellar/stellar-etl-airflow?tab=readme-ov-file#installation-and-setup) documentaiton.
```
Comment (sydneynotthecity) [inline network/hubble/admin-guide/scheduling-and-orchestration/getting-started.mdx:35]: nit: `documentation`

File: network/hubble/admin-guide/scheduling-and-orchestration/getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+## Available DAGs
+
+More information can be found [here](https://github.com/stellar/stellar-etl-airflow/blob/master/README.md#public-dags)
+
+### History Table Export DAG
+
+[This DAG](https://github.com/stellar/stellar-etl-airflow/blob/master/dags/history_tables_dag.py):
+
+- Exports part of sources: ledgers, operations, transactions, trades, effects and assets from Stellar using CaptiveCore
```
Comment (sydneynotthecity) [inline network/hubble/admin-guide/scheduling-and-orchestration/getting-started.mdx:63]: nit: optionally uses CaptiveCore. We may want to be opinionated about that and recommend that running CaptiveCore from Airflow is not an ideal data architecture.

File: network/hubble/admin-guide/scheduling-and-orchestration/getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+<img src={state_table_export} width="300"/>
+
+### DBT Enriched Base Tables DAG
+
+[This DAG](https://github.com/stellar/stellar-etl-airflow/blob/master/dags/dbt_enriched_base_tables_dag.py)
+
+- Creates the DBT staging views for models
+- Updates the enriched_history_operations table
+- Updates the current state tables
+- If found any warnings, it sends a Slack notification about what table has a warning, the time and date it ocurred.
```
Comment (sydneynotthecity) [inline network/hubble/admin-guide/scheduling-and-orchestration/getting-started.mdx:84]: this sentence sounds awkward. would reword

**Review by sydneynotthecity — COMMENTED** [review]
This is awesome! 👏 This greatly improves the Hubble docs, thank you.

Some suggestions and nits on wording but as-is, this is a great first pass

File: network/hubble/admin-guide/data-curation/getting-started.mdx
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/data-curation/getting-started.mdx:1]: Hmm I think it does make sense to document both. Below already does option 2.
I'll add another section like `Advanced Usage - Importing as a dbt package` in this getting-started doc

File: network/hubble/admin-guide/data-curation/overview.mdx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Overview"
+sidebar_position: 0
+---
+
+Data curation in Hubble is done through [stellar-dbt-public](https://github.com/stellar/stellar-dbt-public). stellar-dbt-public transforms base Stellar network data from BigQuery datasets and tables into aggregates and more user analytics friendly tables.
```
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/data-curation/overview.mdx:6]: Oh no my dyslexia is acting up again lol

File: network/hubble/admin-guide/scheduling-and-orchestration/architecture.mdx
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/scheduling-and-orchestration/architecture.mdx:1]: Do we want to mention that here or in our `data-curation/architecture` section?

File: network/hubble/admin-guide/source-system-ingestion/architecture.mdx
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/source-system-ingestion/architecture.mdx:1]: Oh yeah I like that. We should update that diagram though

File: network/hubble/admin-guide/source-system-ingestion/architecture.mdx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+import stellar_etl_arch from '/img/hubble/stellar_etl_architecture.png';
+
+## Architecture Overview
+
+<img src={stellar_etl_arch} width="300"/>
+
+In general stellar-etl runs by:
+
+* Accepting an export command to export data between a start and end ledger
```
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/source-system-ingestion/architecture.mdx:14]: I updated this but not sure if it's what you wanted

File: network/hubble/admin-guide/source-system-ingestion/getting-started.mdx [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+go build
+```
+
+### Run stellar-etl
+
+* A `stellar-etl` executable should have been created in your stellar-etl repo
+* Example stellar-etl command:
+
+```
+./stellar-etl export_ledgers -s 10 -e 11
```
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/source-system-ingestion/getting-started.mdx:47]: No more build docker image; run docker image testing for you anymore lol

File: network/hubble/admin-guide/source-system-ingestion/getting-started.mdx [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+{"base_fee":100,"base_reserve":100000000,"closed_at":"2024-02-06T17:34:17Z","failed_transaction_count":0,"fee_pool":0,"id":47244640256,"ledger_hash":"5b9ac11c6040f4e2fa6a120b3dee9a4b338b7a25bcb8437dab0c0a5c557a41f5","ledger_header":"AAAAAPfImzXFD3TcaerNndqOnsnxrza2opKLd2GcG+tfXKjUK858NP5gM0pneHF0nRowsJBAzMwWDx0+tmbYIZkIT+8AAAAAZcJtmQAAAAAAAAABAAAAANVyadliUPdJbQeb4ug1Ejbv/+jTnC4Gv6uxQh8X/GccAAAAQDhZKPKBdeD4Sthcu+EsuzEtSyiXzXkHboOsgYT1tuV/juZyKqgrsVmg+RmMoRun+NKCdcB8LV9gaehiFm+XDgnfP2GYBKkv20BXGS3EPddI6neK3FK8SYzoBSTAFLgRGfBr+YHFQTIEJ0Y81WEOYClgyjOER8vd4qMQb3gM9nRvAAAACw3gtrOnZAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABkBfXhAAAAAGQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=","max_tx_set_size":100,"operation_count":0,"previous_ledger_hash":"f7c89b35c50f74dc69eacd9dda8e9ec9f1af36b6a2928b77619c1beb5f5ca8d4","protocol_version":0,"sequence":11,"soroban_fee_write_1kb":0,"successful_transaction_count":0,"total_coins":1000000000000000000,"transaction_count":0,"tx_set_operation_count":"0"}
+```
+
+## stellar-etl Commands
```
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/source-system-ingestion/getting-started.mdx:57]: I don't think so. I feel like it'd be rare for a general user to use any of those flags. Advanced users can go look at the repo README or --help

File: network/hubble/admin-guide/visualization/getting-started.mdx (hunk 2 above)
Comment (chowbao) [reply network/hubble/admin-guide/visualization/getting-started.mdx:28]: Lol I had to

**Review by sydneynotthecity — APPROVED** [review]
Two very small feedbacks on the new content but otherwise, let's ship it! 🚢

File: network/hubble/admin-guide/data-curation/getting-started.mdx [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+
+```
+dbt deps
+```
+
+### Running dbt
+
+* There are many useful commands that come with dbt which can be found in the [dbt documentation](https://docs.getdbt.com/reference/dbt-commands#available-commands)
+* Most of stellar-dbt-public will want to use the `dbt build` command which will `run` the model and `test` the model table output
+* The first time running stellar-dbt-public you will want to run the following to create the tables
```
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/data-curation/getting-started.mdx:122]: I added using --full-refresh as optional

File: network/hubble/admin-guide/data-curation/getting-started.mdx [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+### Install required dbt packages
+
+```
+dbt deps
+```
+
+### Running dbt
+
+* There are many useful commands that come with dbt which can be found in the [dbt documentation](https://docs.getdbt.com/reference/dbt-commands#available-commands)
+* Most of stellar-dbt-public will want to use the `dbt build` command which will `run` the model and `test` the model table output
```
[the remark this answers is not in the document]
Comment (chowbao) [reply network/hubble/admin-guide/data-curation/getting-started.mdx:121]: Yeah I like that better
